understanding/20/use-of-color.html

```diff
@@ -167,8 +167,12 @@ <h2>Examples of Use of Color</h2>
                element. A legend shows the color and number for each type of element. Sighted users who
                cannot perceive all the color differences can still understand the image by relying on
                the numbers.</dd>
+         <dt>Focus on links</dt>
+         <dd>A page has text links in blue that have a contrast of 3:1 to the surrounding text. When these links receive keyboard focus, they are shown in a darker color. In addition, they are underlined, so the change of color is not the only indicator of focus.</dd>
```

Comment:
probably need to clarify how much darker the darker color is - i.e. if the difference between the non-focused blue and focused blue is 3:1 or greater, then the underline isn't necessary since a difference in 3:1 or higher counts as an additional visual indication (and the extra underline wouldn't be needed)